WalletWasabi.Fluent/ViewModels/Dialogs/ShuttingDownViewModel.cs

```diff
@@ -10,27 +12,35 @@ public partial class ShuttingDownViewModel : RoutableViewModel
 {
 	private readonly ApplicationViewModel _applicationViewModel;
 	private readonly bool _restart;
+	private readonly CoinJoinManager _coinJoinManager;
 
 	private ShuttingDownViewModel(ApplicationViewModel applicationViewModel, bool restart)
 	{
 		_applicationViewModel = applicationViewModel;
 		_restart = restart;
-		NextCommand = CancelCommand;
+		_coinJoinManager = Services.HostedServices.Get<CoinJoinManager>();
+		NextCommand = ReactiveCommand.CreateFromTask(
+			async () =>
+			{
+				await _coinJoinManager.RestartAbortedCoinjoinsAsync();
+				Navigate().Clear();
+			});
 	}
 
 	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
 	{
-		Observable
-			.Interval(TimeSpan.FromSeconds(3))
-			.ObserveOn(RxApp.MainThreadScheduler)
-			.Subscribe(_ =>
-			{
-				if (_applicationViewModel.CoinJoinCanShutdown())
-				{
-					Navigate().Clear();
-					_applicationViewModel.Shutdown(_restart);
-				}
-			})
-			.DisposeWith(disposables);
+		RxApp.MainThreadScheduler.Schedule(async () => await _coinJoinManager.SignalToStopCoinjoinsAsync());
+
+		Observable.Interval(TimeSpan.FromSeconds(3))
```

Comment:
Please do not reformat code that is not changed.

Reply:
It was the CodeMaid, I guess. 
Anyway, I only added the LOC32.

Reply:
I will check what's going on.

Reply:
Shit, it wasn't CodeMaid. Sorry then, I have no idea why these lines got changed.